MinPlatformPkg/Test/Library/TestPointLib/MmTestPointCommunication.c

```diff
@@ -207,45 +207,42 @@ MmiHandlerTestPointCopyData (
 **/
 VOID
 MmTestPointMmiHandlerGetDataByOffset (
-  IN MMI_HANDLER_TEST_POINT_PARAMETER_GET_DATA_BY_OFFSET     *MmiHandlerTestPointParameterGetDataByOffset
+  IN VOID     *MmiHandlerTestPointParameterGetDataByOffset
   )
 {
-  MMI_HANDLER_TEST_POINT_PARAMETER_GET_DATA_BY_OFFSET    MmiHandlerTestPointGetDataByOffset;
+  MMI_HANDLER_TEST_POINT_PARAMETER_GET_DATA_BY_OFFSET    *MmiHandlerTestPointGetDataByOffset;
   VOID                                                   *Data;
   UINTN                                                  DataSize;
   EFI_STATUS                                             Status;
 
   Data = NULL;
 
-  CopyMem (
-    &MmiHandlerTestPointGetDataByOffset,
-    MmiHandlerTestPointParameterGetDataByOffset,
-    sizeof(MmiHandlerTestPointGetDataByOffset)
-    );
+  MmiHandlerTestPointGetDataByOffset = (MMI_HANDLER_TEST_POINT_PARAMETER_GET_DATA_BY_OFFSET *)MmiHandlerTestPointParameterGetDataByOffset;

```

Comment:
It would be best to check if the parameter `MmiHandlerTestPointParameterGetDataByOffset` is null here. I know it's being passed from code that already checks but its more robust if this function has its own parameter validation without relying on outside functionality.

Reply:
Added.